src/server.c

```diff
@@ -3136,7 +3136,7 @@ void resetServerStats(void) {
 }
 
 /* Make the thread killable at any time, so that kill threads functions
- * can work reliably (default cancelability type is PTHREAD_CANCEL_DEFERRED).
+ * can work reliably (default cancellability type is PTHREAD_CANCEL_DEFERRED).
```

Comment:
i'm not certain. anyone happens to know?

Reply:
I use Clion read the code. Sometimes the editor will hit me. For those unfamiliar words, I always go to Google...
https://translate.google.com/?sl=en&tl=zh-CN&text=cancelability%20&op=translate

Reply:
This one i check again. The spell checker also indicate it

Reply:
Both are correct spellings. Here's an example of documentation which users _cancelability_ with one L: https://linux.die.net/man/3/pthread_setcanceltype (there are others).
I looked at codespell's blame log and didn't find any convincing argument for why they added this as a typo.
I suggest reverting this fix, adding it for now to a whitelist and submitting a fix to codespell to remove this as a typo.

Reply:
Oh i see. alerady updated. Will submitting a fix to codespell